crates/nu-command/src/platform/input/list.rs

```diff
@@ -235,6 +251,13 @@ impl Command for InputList {
                 example: r#"[Banana Kiwi Pear Peach Strawberry] | input list --index"#,
                 result: None,
             },
+            Example {
+                description: "Choose an item from a table using a column as display value",
+                example: r#"[[name price]; [Banana 12] [Kiwi 4] [Pear 7]] | input list -d name"#,
+                result: Some(Value::test_record(
+                    record!("name" => Value::test_string("Kiwi"), "price" => Value::test_int(4)),
```

Comment:
This will have to change to `result: None` I believe. I don't think the CI allows one to choose an option during a run.

Reply:
Ah right, I always forget examples get automatically tested. I'll switch it to `None`.